ngraph/test/models/onnx/mod_sign.prototxt

```diff
@@ -6,18 +6,13 @@ graph {
     input: "B"
     output: "Y"
     op_type: "Mod"
-    attribute {
-      name: "fmod"
-      i: 1
-      type: INT
-    }
```

Comment:
Was it necessary to modify an old model?
Why did you modify it instead of create a new one?

Reply:
I didn't change the old model, but its name to a more meaningful one: mod_sign_fmod.prototxt

BTW. Note that I have changed the type from int64 to int32 because onnx_mod_sign test (now: onnx_mod_sign_fmod) was disabled on the CPU for reasons unrelated to the onnx importer.